### DISCUSSION TITLE
Adding Polybench to gem5-resources

### DISCUSSION THREAD

**Discussion by Sir-NoChill** [gem5/gem5#1442] [gem5-dev]
I'm polling to see if there would be any interest in adding polybench to the gem5-resources site. I'm currently creating the suite for course use and wanted to know if I should have putting it in the gem5-resources on my radar.

**Comment by mattsinc** [comment]
@Sir-NoChill I'm assuming you mean the GPU Polybench tests?  If so, you are definitely welcome to add them.  Ultimately, it's important that we test the additions (so they don't break over time) and add appropriate documentation so users understand how to use them.  Good luck!
